counterpoll/main.py

```diff
@@ -504,6 +504,36 @@ def wredport_disable(ctx):
     wred_port_info = {}
     wred_port_info['FLEX_COUNTER_STATUS'] = 'disable'
     ctx.obj.mod_entry("FLEX_COUNTER_TABLE", "WRED_ECN_PORT", wred_port_info)
+# SRv6 counter commands
+@cli.group()
+@click.pass_context
+def srv6(ctx):
+    """ SRv6 counter commands """
+    ctx.obj = ConfigDBConnector()
+    ctx.obj.connect()
+
+@srv6.command()
+@click.pass_context
+@click.argument('poll_interval', type=click.IntRange(100, 30000))
```

Comment:
Do we really want to support 0.1-second polling interval? Will it overwhelm the system?

Reply:
I made it 0.1 since most of the other counters do so. Now, on second thought, I think the 1000 should be min, I'll check the requirements